linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml

```diff
@@ -34,6 +34,7 @@ identifiers:
     cce@rhel9: CCE-83453-1
     cce@sle12: CCE-83001-8
     cce@sle15: CCE-83260-0
+    cce@slmicro5: CCE-93601-3
```

Comment:
Will there be a CCE pool file, similar to `shared/references/cce-sle15-avail.txt`?

Reply:
For sure there is a pending PR #12109 on me for that, that I will review/merge as soon as the current one is approved